config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -22,7 +22,7 @@ presubmits:
   - name: pull-kubebuilder-e2e-k8s-1-25-0
     decorate: true
     always_run: true
-    optional: true
+    optional: false
```

Comment:
Hi @coriaedu,

That is great. Thank you.
But we actually need to be all mandatory. 
Is possible to change it out?

Reply:
I was wondering about the other k8s versions too... I will change that

Reply:
@camilamacedo86 done